### PR TITLE
WINDUP-885  Upgrade windup-maven-plugin to latest WindUp core version

### PR DESCRIPTION
**Pull request by OndraZizka**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -29,240 +29,261 @@
...
-          <version>0.9.0-M1</version>
-      </dependency>
-      <dependency>
-          <groupId>com.thinkaurelius.titan</groupId>
-          <artifactId>titan-core</artifactId>
-          <version>0.9.0-M1</version>
-      </dependency>
+    <dependencies>
 
+        <dependency>
```
Comment (jsight) [inline pom.xml]: This version of titan is incorrect. We use 0.5.4

File: pom.xml [hunk 2]
```diff
@@ -29,240 +29,261 @@
...
-      <dependency>
-          <groupId>org.jboss.weld</groupId>
-          <artifactId>weld-core</artifactId>
-          <version>2.2.8.Final</version>
-      </dependency>
+        <dependency>
+            <groupId>org.jboss.weld</groupId>
+            <artifactId>weld-api</artifactId>
+            <version>2.2.Final</version>
+        </dependency>
```
Comment (mareknovotny) [inline pom.xml]: wrong indentation?

File: src/it/simple-it/pom.xml [hunk 3]
```diff
@@ -11,9 +11,9 @@
 
   <properties>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-      <windup.version>2.0.0-SNAPSHOT</windup.version>
-      <forge.version>2.12.1.Final</forge.version>
-      <inputDirectory>/Users/samueltauil/Development/windup/jee-example-app-1.0.0.ear</inputDirectory>
+      <windup.version>2.5.0.CR1</windup.version>
+      <forge.version>2.20.1.Final</forge.version>
+      <inputDirectory>${project.basedir}/../Applications/public/jee-example-app-1.0.0.ear</inputDirectory>
```
Comment (mareknovotny) [inline src/it/simple-it/pom.xml]: is this user specific path good to have here? I guess we need a relative path you can use without pulling/checking out another resource(s)

File: pom.xml [hunk 4]
```diff
@@ -1,27 +1,27 @@
...
+    <name>Windup Maven Plugin</name>
 
-  <name>Windup Maven Plugin</name>
+    <url>http://windup.jboss.org</url>
 
-  <url>http://windup.jboss.org</url>
-
-  <properties>
-    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-    <windup.version>2.4.1.Final</windup.version>
```
Comment (mareknovotny) [inline pom.xml]: we probably would like to have here more up-to-date version 2.5.0.CR1/Final or 2.5.0-SNAPSHOT while Final is not ready. What do you think?

Comment (jsight) [inline pom.xml]: Yes, it looks like it is set to 2.5.0-SNAPSHOT?

File: src/it/simple-it/pom.xml (hunk 3 above)
Comment (jsight) [inline src/it/simple-it/pom.xml]: That looks broken to me. Having said that, I am not sure that this test is even running. PRs welcome. :)

Comment (samueltauil) [inline src/it/simple-it/pom.xml]: probably I'm guilty in this PR because I merged that without test :/

Comment (jsight) [inline src/it/simple-it/pom.xml]: No worries... at least the functionality works. :)

**Pull request merged**